:mod:`turbo_response.exceptions`
================================

.. py:module:: turbo_response.exceptions


Module Contents
---------------

Classes
~~~~~~~

.. autoapisummary::

   turbo_response.exceptions.InvalidTurboStream
   turbo_response.exceptions.InvalidTurboFrame



.. py:class:: InvalidTurboStream

   Bases: :class:`django.core.exceptions.ValidationError`


.. py:class:: InvalidTurboFrame

   Bases: :class:`django.core.exceptions.ValidationError`


